modules/nf-cid/src/main/nextflow/data/cid/fs/CidPath.groovy

```diff
@@ -385,3 +408,102 @@ class CidPath implements Path, RealPathAware {
     }
 
 }
+
+class ResultsSeekableByteChannel implements SeekableByteChannel {
```

Comment:
Missing `@CompileStatic`. Also may be convenient to keep in its own source file

Reply:
Fixed